src/tools/rbd_mirror/Replayer.cc

```diff
@@ -228,26 +83,22 @@ class MirrorStatusWatchCtx {
 };
 
 Replayer::Replayer(Threads *threads, std::shared_ptr<ImageDeleter> image_deleter,
-                   RadosRef local_cluster, const peer_t &peer,
-                   const std::vector<const char*> &args) :
+                   RadosRef local_cluster, int64_t local_pool_id,
+                   const peer_t &peer, const std::vector<const char*> &args) :
   m_threads(threads),
   m_image_deleter(image_deleter),
   m_lock(stringify("rbd::mirror::Replayer ") + stringify(peer)),
```

Comment:
Right now the Replayer is still tied to a single peer -- so that would provide a unique lock name.  Eventually when we support multiple peers, this class might morph into ```PoolReplayer``` which is tied to the local pool with multiple ```PeerReplayers``` associated for each remote peer.